uuid_test.go

```diff
@@ -29,6 +29,28 @@ func TestGenerateUUID(t *testing.T) {
 	}
 }
 
+func TestGenerateUUIDByReader(t *testing.T) {
+	prev, err := GenerateUUIDByReader(rand.Reader)
+	if err != nil {
+		t.Fatal(err)
+	}
+	for i := 0; i < 100; i++ {
+		id, err := GenerateUUIDByReader(rand.Reader)
```

Comment:
Not sure I'd bother with the loop, you're basically testing that rand.Reader works here more than anything.  I'd also worry about how slow this might make test execution in circleci, since we don't know how well they're fixed for entropy.  If you keep the loop I suggest something much smaller than 100.

Reply:
I think this can come out